docs/jwst/cube_build/index.rst

```diff
@@ -10,4 +10,12 @@ Cube Building
    examples.rst
    reference_files.rst
    
-.. automodapi:: jwst.cube_build
+.. automodapi:: jwst.cube_build.cube_build_step
+.. automodapi:: jwst.cube_build.data_types
```

Comment:
Do we want all these lower level functions in the main docs?  I would suspect all we need is `cube_build_step` for the docs.  Or do we expect users to need to call these lower level functions?

Reply:
I was not sure if every module had to be listed in index.rst or just the main code.
I will make it just cube_build_step